doc/python_reference.rst

```diff
@@ -184,6 +186,15 @@ Functions:
    write_surface
    write_trans
 
+.. currentmodule:: mne.io
+
+:py:mod:`mne.io`:
```

Comment:
why do you have this again? did you check how it rendered?

https://2220-1301584-gh.circle-artifacts.com/0//home/ubuntu/mne-python/doc/_build/html/python_reference.html